docs/DeveloperGuide.adoc

```diff
@@ -77,6 +77,30 @@ Use case ends.
 +
 Use case resumes at step 2.
 
+=== Use case: Rename tag
+
+*MSS*
+
+. User requests to list tags
+. AddressBook shows a list of tags
+. User requests to rename a specific existing tag in the list
+. AddressBook requests for new tag name with the user
+. AddressBook confirms the change of tag name with the user
```

Comment:
Ouch! Remember, it was mentioned that the use case should show the interaction between the user and the system?
After the above step, the `User` should do some operation.